src/devices/cpu/i386/i386.cpp

```diff
@@ -3944,9 +3959,76 @@ void i386_device::device_reset()
 
 	m_CPL = 0;
 
+	m_auto_clear_RF = true;
+
 	CHANGE_PC(m_eip);
 }
 
+inline void i386_device::dri_changed()
```

Comment:
Since that's called from the notifier, could you add a quick return when no breakpoint is requested?